docs/package.json

```diff
@@ -43,15 +43,15 @@
     "@mui/system": "workspace:^",
     "@mui/types": "workspace:^",
     "@mui/utils": "workspace:^",
-    "@mui/x-charts": "6.19.8",
+    "@mui/x-charts": "7.3.2",
     "@mui/x-data-grid": "7.3.1",
     "@mui/x-data-grid-generator": "7.3.1",
     "@mui/x-data-grid-premium": "7.3.1",
     "@mui/x-data-grid-pro": "7.3.1",
-    "@mui/x-date-pickers": "6.19.9",
-    "@mui/x-date-pickers-pro": "6.19.9",
+    "@mui/x-date-pickers": "7.3.2",
+    "@mui/x-date-pickers-pro": "7.3.2",
     "@mui/x-license-pro": "6.10.2",
```

Comment:
Consider removing it.
The package does not seem to be used anywhere explicitly and has already been renamed to `@mui/x-license`.
It is a direct dependency of the paid packages and if there are no direct references to it in the project, I don't see a reason to have it explicitly listed. 🙈 
```suggestion
```

Reply:
We have

```
import { LicenseInfo } from '@mui/x-data-grid-pro';
```

Which IMHO is a bad pattern now that we use the license key for several X commercial packages
I would prefer to install `@mui/x-license` and set the license from it instead of setting from the grid (which is deprecated in #11956)

Reply:
Great point. 👌 
However, IMHO, this could also be a separate PR as well, because it does not seem directly related to what is being done here. 🤔

Reply:
Re-requesting reviews just to make sure I didn't mess something up 🙈